src/components/Navbar.js

```diff
@@ -0,0 +1,105 @@
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { logout } from '../store/user_reducer';
+import logo from '../img/logo3.png';
+
+function Navbar() {
+  const dispatch = useDispatch();
+  const [burger, setBurger] = useState(true);
+  // const [mobile, setmobile] = useState(false);
```

Comment:
- [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the app. 💪